### PR TITLE
OOP school library: set up associations

### PR DESCRIPTION
**Pull request by CarlosE-Feliz**
- Set up associations between classes and objects. ✔

### CODE REVIEW

File: classroom.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+attr_accessor :label
+attr_reader :students
+
+class Classroom
```
Comment (hiromataba) [inline classroom.rb]: - [x] Please put your accessors methods  inside the `Classroom` class to access them inside your class

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi :wave: @CarlosE-Feliz , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
### To highlight :1st_place_medal: 
- No linter errors :heavy_check_mark: 
- Descriptive  README file :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: classroom.rb (hunk 1 above)
Comment (CarlosE-Feliz) [reply classroom.rb]: Oh! Thanks, I did not notice it!

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @CarlosE-Feliz, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: classroom.rb (hunk 1 above)
Comment (hiromataba) [reply classroom.rb]: You are welcome @CarlosE-Feliz

**Pull request merged**